examples/blog/src/pages/index.astro

```diff
@@ -3,6 +3,7 @@
 import BaseHead from '../components/BaseHead.astro';
 import BlogHeader from '../components/BlogHeader.astro';
 import BlogPostPreview from '../components/BlogPostPreview.astro';
+import '../styles/blog.css';
```

Comment:
Two nits: 
1. could this just be moved to a single reference in BaseHead?
2. based on https://github.com/withastro/rfcs/blob/main/active-rfcs/0000-style-unification.md we said we'd recommend `<style>@import '../styles/blog.css';</style>` over an ESM import. Mind updating to that syntax?

Reply:
done